packages/rml/rml.1.09.06/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+homepage: "http://reactiveml.org"
+maintainer: "devel@reactiveml.org"
+bug-reports: "bugs@reactiveml.org"
+build: [
+["./configure" "--prefix" "%{prefix}%"]
+[make]
+]
+install: [
+ [make "install"]
+]
+remove: [
+ ["./configure" "--prefix" "%{prefix}%"]
+ [make "uninstall"]
+]
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "num"
+  "stdlib-shims"
```

Comment:
oh it seems you are already using `stdlib-shims`. From the error message it seems you forgot to use it when compiling